### PR TITLE
Removal of antD reference from SiderMenu

### LINKED ISSUE #181: Remove Ant Design from SiderMenu component

### PR DESCRIPTION
**Pull request by anishaswain**
- Removes reference of SiderMenu which used AntD (SiderMenu component isn't required anymore)
- Changes NavigationDrawer component's unit test to remove reference of SiderMenu in it( looks like it was a bug)

### CODE REVIEW

**Review by aquibbaig — DISMISSED** [review]
Looks great! 👍 
Can you please remove the padding-top on `pf-c-page__sidebar-body`, it would be great, thanks!

**Comment by anishaswain** [comment]
> Looks great! 👍
> Can you please remove the padding-top on `pf-c-page__sidebar-body`, it would be great, thanks!

@aquibbaig which page are you referencing here?

**Review by dbutenhof — DISMISSED** [review]
Ah; so basically this was copied from `SiderMenu` but the unit test still (apparently successfully) tests `SiderMenu` which means `NavigationDrawer` wasn't being (unit) tested before?

**Comment by anishaswain** [comment]
> Ah; so basically this was copied from `SiderMenu` but the unit test still (apparently successfully) tests `SiderMenu` which means `NavigationDrawer` wasn't being (unit) tested before?

`SiderMenu` is not referenced anywhere is a current code. Somehow we missed to delete the component. And unit test  of `NavigationDrawer` was by mistake exporting and using `SiderMenu` component ,so it wasn't throwing error as it is just snapshot testing.

**Comment by aquibbaig** [comment]
> > Looks great! 👍
> > Can you please remove the padding-top on `pf-c-page__sidebar-body`, it would be great, thanks!
> 
> @aquibbaig which page are you referencing here?

@anishaswain The padding on top here
Screenshot 2021-08-16 at 5 38 39 PM

**Comment by anishaswain** [comment]
> > > Looks great! 👍
> > > Can you please remove the padding-top on `pf-c-page__sidebar-body`, it would be great, thanks!
> > 
> > 
> > @aquibbaig which page are you referencing here?
> 
> @anishaswain The padding on top here


@aquibbaig this style is by default in PatternFly CSS file. We haven't used any custom CSS here, so I am not sure if we should use another css file just to address a single custom style change. (And not a fan of inline styles)

**Comment by aquibbaig** [comment]
@anishaswain I think any of the solutions work for me, this padding looks a bit odd to me

**Comment by anishaswain** [comment]
> @anishaswain I think any of the solutions work for me, this padding looks a bit odd to me

@aquibbaig could you check it now.

**Review by dbutenhof — DISMISSED** [review]

File: src/components/NavigationDrawer/index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React, { PureComponent } from 'react';
 import { Link } from 'dva/router';
 import { PageSidebar, Nav, NavList, NavItem } from '@patternfly/react-core';
 import getMenuData from '../../common/menu';
+import './index.less';
```
Comment (aquibbaig) [inline src/components/NavigationDrawer/index.js:5]: There is no `index.less` now and the padding is still there

Comment (anishaswain) [reply src/components/NavigationDrawer/index.js:5]: @aquibbaig I somehow missed adding the file to commit. Please re-review it now.

**Review by aquibbaig — APPROVED** [review]

**Pull request merged**